moped-editor/src/components/GridTable/FiltersCommonOperators.js

```diff
@@ -168,4 +168,32 @@ export const FiltersCommonOperators = {
     type: "array",
     envelope: "false",
   },
+  council_districts_array_is_null: {
+    operator: "_is_null",
+    label: "No",
+    description: "Project has no mapped components in council districts",
+    type: "array",
+    envelope: "true",
+  },
+  council_districts_array_is_not_null: {
+    operator: "_is_null",
+    label: "Yes",
+    description: "Project has components in council districts",
+    type: "array",
+    envelope: "false",
+  },
+  council_districts_array_contains: {
+    operator: "_contains",
+    label: "Contains",
+    description: "Project has components in a given council district",
+    type: "array",
+    envelope: false,
+  },
+  council_districts_array_is: {
+    operator: "_eq",
+    label: "Is",
+    description: "Project has components in, and only in, specific council districts",
+    type: "array",
+    envelope: false,
+  },
```

Comment:
I am a little queasy here about making these operators that are specific to the council districts, but I couldn't figure out a way to make the existing array-type, field specific operator's descriptions work with the council district's needs. If you look just above these declarations, you can see the existing ones I'm referring to.

Reply:
Column specific operators sound good to me too. One of the challenges of refactoring some of this was the use of more general terms like "special case" so I am all for explicit, column-specific naming if we can't find a way to generalize. 🙏